src/lib/components/Components.js

```diff
@@ -0,0 +1,207 @@
+import {
```

Comment:
I would also split this file into 
`likes`
`posts`
`users`
etc

Reply:
Done